lib/browser/api/ipc-main.ts

```diff
@@ -1,8 +1,44 @@
 import { EventEmitter } from 'events'
+import { IpcMainEvent } from 'electron'
 
-const emitter = new EventEmitter()
+class IpcMain extends EventEmitter {
+  _invokeHandlers: {[channel: string]: (e: IpcMainEvent, ...args: any[]) => void};
+  constructor () {
```

Comment:
This constructor is not required, we can initialize in the property declaration above